data/countries/PY.yaml

```diff
@@ -12,6 +15,14 @@ holidays:
     days:
       01-01:
         _name: 01-01
+      03-01:
+        disable:
+          - '2017-03-01'
```

Comment:
After reading through the YAML specification, this appeared to be a the preferred method of changing the date on a one-off year. Please let me know if there is a better way.

Reply:
This is the only method. The reason is that some calendars offer more than one date per gregorian year.

Reply:
Ok, that makes sense. However, suppose that the Paraguayan government adjusted this holiday in 2020 by a day or two. How would that be represented here?

Reply:
Please check https://github.com/commenthol/date-holidays/blob/5deac678e61afbe2b72d2b7005501c17291f6e15/data/countries/GY.yaml#L44

Reply:
Oh, nice! Tuples.